File: locus-api-android-sample/src/main/java/com/asamm/locus/api/sample/pages/PageWelcomeFragment.java
package com.asamm.locus.api.sample.pages;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.asamm.locus.api.sample.R;

import androidx.fragment.app.Fragment;

/**
 * Created by menion on 29/08/2016.
 * This code is part of Locus project from Asamm Software, s. r. o.
 */
public class PageWelcomeFragment extends Fragment {

	@Override
	public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
		return inflater.inflate(R.layout.page_welcome, container, false);
	}
}

File: locus-api-android-sample/src/main/java/com/asamm/locus/api/sample/pages/PageWelcomeFragment.kt
/**
 * Created by menion on 29/08/2016.
 * This code is part of Locus project from Asamm Software, s. r. o.
 */
package com.asamm.locus.api.sample.pages

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.asamm.locus.api.sample.R

class PageWelcomeFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.page_welcome, container, false)
    }
}
